README.md

```diff
@@ -24,6 +24,12 @@ The following utilities are prerequisites to installing F´:
 - git
 - Python 3.5+ with pip
 
+You can install them using this one-liner command :
+
+```
+sudo apt-get -y install cmake git python3-pip
```

Comment:
- this won't work on RHEL
- generally if you're going to give advice like this, you should tell people to run `apt-get update` before `apt-get install` ...

Reply:
@jsoref For RHEL as per I have checked we can install the same using 
``` 
yum install cmake git python3-pip 
``` 
Also yeah I will add up that change to even do the ``` sudo apt update``` before installing the requirements.